### PR TITLE
save settings when updating

### LINKED ISSUE #60: Remember config
Salut!

I use `SVGOMG` every single day, it is just great! But each time, I have to re-set my config. I don't thing it would be really hard to implement a **cookie system** that would **remember** our config.

Math

### PR DESCRIPTION
**Pull request by psimyn**
fixes #60

Potential issues:

- no opt-out / reset
- a better way to update range slider

### CODE REVIEW

File: src/js/page/ui/settings.js [hunk 1]
```diff
@@ -67,14 +67,32 @@ class Settings extends (require('events').EventEmitter) {
...
+  setSettings(settings) {
+    this._globalInputs.forEach(function(inputEl) {
+      if (inputEl.type == 'checkbox') {
+        inputEl.checked = settings[inputEl.name];
+      }
+      else if (inputEl.type == 'range') {
+        inputEl.value = settings[inputEl.name];
+        var rangeChangeEvent = utils.isIe ? 'change' : 'input';
+        var event = new Event(rangeChangeEvent);
+        inputEl.dispatchEvent(event);
```
Comment (jakearchibald) [inline src/js/page/ui/settings.js]: Why dispatch an event here? I don't think setting `checked` triggers events.

Comment (psimyn) [inline src/js/page/ui/settings.js]: This is only for the range - just updating the input didn't update the material-slider elements so ui was out of sync

File: src/js/page/main-controller.js [hunk 2]
```diff
@@ -155,6 +156,7 @@ class MainController {
     try {
       this._inputSvg = await svgo.load(event.data);
       this._inputFilename = event.filename;
+      await this._loadSettings();
```
Comment (jakearchibald) [inline src/js/page/main-controller.js]: Any objections to doing this on DOM ready? Happy to make that change as I merge

File: src/js/page/ui/settings.js (hunk 1 above)
Comment (jakearchibald) [inline src/js/page/ui/settings.js]: Ahhh gotcha, I'll work around this

File: src/js/page/main-controller.js (hunk 2 above)
Comment (psimyn) [inline src/js/page/main-controller.js]: :+1:

**Comment by jakearchibald** [comment]
It's about time I started a contributors file. What name should I use there?

**Comment by psimyn** [comment]
Simon Knox or psimyn, depending on format

**Pull request merged**